### DISCUSSION TITLE
Bug with Ellispses

### DISCUSSION THREAD

**Discussion by BigAloz** [haplokuon/netDxf#455] [General]
Hello,

Thanks for this excellent software.

I think I may have found a bug with your Entities.Ellipses.

I was trying to debug why the ellipses I was drawing were much bigger than what they are in Autocad.

An comparing data I noticed that the semi-major and semi-minor axis are actually twice the correct values. Show as a diameter rather than as the radius.

Hope this is useful

**Comment by haplokuon** [comment]
That is why those values  in the ellipse class are called major and minor axis and not semi-major and semi-minor axis. There is no bug here.

**Reply by BigAloz** [reply]
Thanks very much.

I just noticed they were twice the value of what Autocad properties were showing for the same ellipse.

If its designed that way then great. I just flagged it any case it had gotten through the cracks
